metricshub-doc/src/site/markdown/connector-design-guide/connector-object-references.md

```diff
@@ -0,0 +1,173 @@
+keywords: guide, connector, connector, object, reference
+description: This page shows how to reference connector objects such as sources, entries, columns and files, etc. inside the YAML document.
+
+# Connector Object References
+
+This page shows how to reference connector objects such as sources, entries, columns and files, etc. inside the YAML document.
+
+* [Source Reference](#Source_Reference)

```

Comment:
In Markdown, the generated identifier for a given title follows a specific format: the title is converted to lowercase, and any spaces are replaced with hyphens. For example, the `#source-reference` identifier is generated from the `Source Reference` title.

[Source Reference](#source-reference)